Classes/Controller/BackendController.php

```diff
@@ -47,7 +49,10 @@ class BackendController extends ActionController
 
     public function __construct(KeyRepository $keyRepository, BackendUserRepository $backendUserRepository)
     {
-        parent::__construct();
+        if (Comparator::greaterThanOrEqualTo(Versions::getVersion('typo3/cms-core'), '10')) {
```

Comment:
why we need  extra dependencies here? 
`  if (version_compare(TYPO3_version, '10.0.0', '>=')) {
`
this does the same job

Reply:
We are using the `Comparator` and `Versions` class also in the `Factory` class: https://github.com/cvc-digital/typo3-webauthn/pull/8/files#diff-730b305699c79dc320b26ffe41591fcaR53-R56

The reason, why we need `Versions` is that `web-auth/webauthn-lib` does not provide any version information.